src/pages/register-a-beacon/activity.tsx

```diff
@@ -398,6 +498,145 @@ const AviationCommercialOptions: FunctionComponent<OptionsProps> = ({
   );
 };
 
+const LandOptions: FunctionComponent<OptionsProps> = ({
```

Comment:
Made activity options radio buttons and moved across from `land-other-activity` page